diesel_tests/build.rs

```diff
@@ -24,13 +24,26 @@ fn connection() -> SqliteConnection {
     SqliteConnection::establish(&database_url).unwrap()
 }
 
+#[cfg(feature = "mysql")]
+use self::diesel::mysql::MysqlConnection;
+#[cfg(feature = "mysql")]
+fn connection() -> MysqlConnection {
+    dotenv().ok();
+    let database_url = ::std::env::var("MYSQL_DATABASE_URL")
```

Comment:
I'm going to apply this pattern to the entire codebase soon. One env var is great for CI, terrible for local development.